tox.ini

```diff
@@ -8,6 +8,8 @@ passenv=
   CI
 
 [testenv:test]
+deps=
+  flask>=1.1,<2
```

Comment:
If you take a look at the circle ci file, you see it does `pip install '.[dev]' tox` to intsall the dependencies.

So as flask is already inclueded in dev dependencies (look at setup.py line 35), it should not be needed.

Reply:
could you send me the Ci where it fails when remove it ?